packages/repo/src/sync.ts

```diff
@@ -4,12 +4,14 @@ import { RepoStorage } from './storage'
 import { DataDiff } from './mst'
 import Repo from './repo'
 import * as verify from './verify'
+import * as util from './util'
 
 export const loadRepoFromCar = async (
   carBytes: Uint8Array,
   storage: RepoStorage,
   verifier: auth.Verifier,
 ): Promise<Repo> => {
+  const car = await util.readCar(carBytes)
```

Comment:
I don't see `car` used anywhere— is this a way to verify the bytes are for a car?  If so, might just be worth dropping a comment.

Reply:
ah nope just missed it on cleanup 👌